client/me/purchases/payment/edit-card-details/index.jsx

```diff
@@ -70,10 +71,10 @@ class EditCardDetails extends PurchaseCardDetails {
 
 				<CreditCardForm
 					apiParams={ this.getApiParams() }
+					createPaygateToken={ createPaygateToken( 'card_update' ) }
```

Comment:
I still forget about this one, nice catch :)
Yes, constructor approach or even outside of the class.